credentials/apps/verifiable_credentials/rest_api/v1/views.py

```diff
@@ -60,39 +72,69 @@ class InitIssuanceView(APIView):
+            msg = _(f"Incomplete required data: ['credential_uuid', 'storage_id']")
+            logger.exception(msg)
+            return Response(msg, status=status.HTTP_400_BAD_REQUEST)
+
+        if not is_valid_uuid(credential_uuid):
+            msg = _(f"Credential identifier must be valid UUID: ['credential_uuid']")
+            logger.exception(msg)
+            return Response(msg, status=status.HTTP_400_BAD_REQUEST)
+
+        issuance_line = CredentialIssuer.init(
+            credential_uuid=credential_uuid,
+            storage_id=storage_id,
+        )
 
-        if not credential_uuid:
-            return Response("Credential uuid is required", status=status.HTTP_400_BAD_REQUEST)
+        deeplink = issuance_line.storage.get_deeplink_url(issuance_line.uuid)
 
-        issuance = CredentialIssuer.create_issuance_request(credential_uuid)
+        init_data = {
+            "deeplink": deeplink,
+            "qrcode": generate_base64_qr_code(deeplink),
+        }
 
-        return Response(
-            {
-                "deeplink": Wallet.create_deeplink_url(issuance.uuid),
-                "qrcode": Wallet.create_qr_code(issuance.uuid),
-                "app_link_android": Wallet.APP_LINK_ANDROID,
-                "app_link_ios": Wallet.APP_LINK_IOS,
-            }
-        )
+        if issuance_line.storage.is_mobile():
+            init_data.update(
+                {
+                    "app_link_android": issuance_line.storage.APP_LINK_ANDROID,
+                    "app_link_ios": issuance_line.storage.APP_LINK_IOS,
+                }
+            )
+
+        return Response(init_data)
 
 
 class IssueCredentialView(APIView):
     """
-    This API endpoint allow requests for VC issuing.
-    POST: /verifiable_credentials/api/v1/credential/issue/{issuance_uuid}
-    Arguments:
-        request: A request to control data returned in endpoint response
-    Returns:
-        response(dict): signed VC document for storing
+    This API endpoint allows requests for VC issuing.
+
+    POST: /verifiable_credentials/api/v1/credential/issue
+
+    Request and response should conform VC API specs:
+    https://w3c-ccg.github.io/vc-api/#issue-credential
     """
 
     authentication_classes = ()
 
+    # permission_classes = (IsAuthenticated,)
     permission_classes = ()
 
     def post(self, request, *args, **kwargs):
-        credential_issuer = CredentialIssuer(request.data, kwargs.get("issuance_uuid"))
-        verifiable_credential_document = credential_issuer.issue()
+        credential_issuer = CredentialIssuer(request.data, kwargs.get("issuance_line_uuid"))
+
+        return Response({"verifiableCredential": credential_issuer.issue()}, status=status.HTTP_201_CREATED)
+
 
-        return Response(verifiable_credential_document)
+class NoWalletView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):
+        return render(
+            request,
+            "no-wallet.html",
```

Comment:
NoWallet template does not exists.
Put this path template `verifiable_credentials/no-wallet.html` instead of `no-wallet.html`

Reply:
Do we actually need `verifiable_credentials` folder in templates? We already in `verifiable_credentials` app